HotorNot/app/src/main/java/com/example/hotornot/fragments/MainScreenFragment.kt

```diff
@@ -30,6 +31,16 @@ class MainScreenFragment : Fragment() {
         binding.btnLeft.setOnClickListener {
             binding.imageView.setImageResource(R.drawable.georgi)
             binding.txtName.text = R.string.georgi.toString()
+
+        }
+    }
+
+    private fun makeBtnDisappear(){
```

Comment:
Please,consider more appropriate for example buttonVisibility() or something like this